docs/Developers/Development_Setup.rst

```diff
@@ -91,30 +91,24 @@ https://gitlab.com/coala/coala-utils/
 Installing from Git
 -------------------
 
-We recommend first installing the latest development snapshot of coala's
-master branch from and all of its dependencies with pip3 using
+We recommend installing coala from the master branch for latest updates
+ and its dependencies with pip3 using
 
 ::
 
     (coala-venv)$ git clone https://github.com/coala/coala
     (coala-venv)$ cd coala
     (coala-venv)$ pip3 install .
+    # This will install coala from the current directory.
     (coala-venv)$ cd -
     (coala-venv)$ git clone https://github.com/coala/coala-bears
     (coala-venv)$ cd coala-bears
     (coala-venv)$ pip3 install .
+    # This will install coala-bears from the current directory.
 
-Once you have forked the repository you would like to modify, you can
-install a repository-backed version of the repository using
-
-::
-
-    (coala-venv)$ pip3 install -e <path/to/forked/repository>
```

Comment:
If you are never gonna use the `-e` tag, you are never gonna be able to test the changes you made locally. Add the `-e` tag to the initial command

Reply:
I dont think it s so. @ishanSrt

Reply:
Its already tested @newbazz 😉. If not then create a new virtual environment and test it yourself 👍

Reply:
It is not that you are never gonna be able to test the changes made ;P

Reply:
??

Reply:
I dont get this @ishanSrt ; the changes do add `-e`.

Reply:
@jayvdb this suggested change is from earlier when `-e` was not being used by the OP. Since the deleted lines are the same, it still shows here